tox.ini

```diff
@@ -78,7 +78,7 @@ deps =
     mindeps: Automat == 0.8.0
     mindeps: hyperlink == 17.1.1
     mindeps: attrs == 21.3.0
-    mindeps: typing_extensions == 3.10.0
+    mindeps: typing_extensions == 4.0.0
```

Comment:
A thought for future work here; `typing_extensions` really ought to be an optional dependency that we don't use on my recent Pythons where it's not required. 3.11 has `typing.Self` for example.

Reply:
Yeah but then it's horrible if this else that import